lib/msal-browser/docs/device-bound-tokens.md

```diff
@@ -0,0 +1,53 @@
+# Acquiring Device Bound Tokens using Web Account Manager (WAM) on Windows
+
+MSAL.js supports acquiring tokens from the Web Account Manager (WAM) on Windows. These tokens are bound to the device they were acquired on and are not cached in the browser's localStorage or sessionStorage.
+
+## Supported Environment
+
+This feature is currently only supported in the following environment:
+
+- A machine running a Windows build that supports the feature (more to come on this)
+- Chrome and Edge browsers or Teams
+- [Windows Accounts extension](https://chrome.google.com/webstore/detail/windows-accounts/ppnbnpeolgkicgegkbkbjmhlideopiji) is installed if using Chrome or Edge
+- App must be hosted on `https`
+
+Additionally, this feature is currently only supported for Work and School Accounts
+
+## Enable the feature in MSAL.js
+
+In order to enable this feature in MSAL.js set the `allowNativeBroker` flag to true in your configuration object like so:
+
+```javascript
+const msalConfig = {
+    auth: {
+        clientId: "insert-clientId"
+    },
+    system: {
+        allowNativeBroker: true
+    }
+};
+```
+
+Additionally, you will need to call and await the new `initialize` API before invoking any other MSAL.js API.
+
+```javascript
+const pca = new PublicClientApplication(msalConfig);
+
+// Initialize will establish a connection with the browser extension, if present
+await pca.initialize();
+
```

Comment:
I would show invoking `handleRedirectCallback` so that it is clear what the timing should be.

Reply:
Done